astropy/units/equivalencies.py

```diff
@@ -617,19 +639,19 @@ def convert_K_to_Jy(x_K):
             return (x_K * beam / factor)
 
         return Equivalency([(astrophys.Jy, si.K, convert_Jy_to_K, convert_K_to_Jy),
-                            (astrophys.Jy/astrophys.beam, si.K, convert_Jy_to_K, convert_K_to_Jy),],
-                           "brightness_temperature", {'frequency': frequency, 'beam_area': beam_area})
+                            (astrophys.Jy/astrophys.beam, si.K, convert_Jy_to_K, convert_K_to_Jy), ],  # noqa: E501
```

Comment:
Not sure why one wouldn't just remove the `,` before the `]` instead of adding a space, since it is a list.